src/index.js

```diff
@@ -73,13 +69,10 @@ function install (Vue) {
 
 function installEvents (vm) {
   vm.$on('modal:shouldShow', function (name) {
-    this.$broadcast('modal:show', name)
-  })
-  vm.$on('retriever:fetched', function (el) {
-    vm.$compile(el);
+  //  this.$broadcast('modal:show', name)
```

Comment:
I don't think we should comment this out, the migration helper will not be able to catch these if we forget to remove it subsequently. We should remove them in the `Modal` and `Trigger`'s migration PR instead.